python/ray/autoscaler/_private/kubernetes/config.py

```diff
@@ -30,6 +31,10 @@ def not_found_msg(resource_type, name):
         resource_type, name)
 
 
+def not_checking_msg(resource_type, name):
+    return "not checking if {} '{}' exists".format(resource_type, name)
```

Comment:
I would expand on this error message and say "Not checking if namespace exists due to insufficient permissions, the command may fail if it doesn't."